hack/ds.yaml

```diff
@@ -16,7 +16,7 @@ spec:
       memory: 1024Mi
   storage: 5Gi
   # StorageClassName for the PVC. This should be very fast SSD disk for the directory server data
-  storageClassName: fast
+  storageClassName: standard
```

Comment:
Why to standard?

Reply:
this is what minikube comes out of the box with, and I believe GKE also creates "standard" as well

Trying to make this easy for users - since they may not know how to create the storageclass